### DISCUSSION TITLE
Welcome to kruise Discussions!

### DISCUSSION THREAD

**Discussion by FillZpp** [openkruise/kruise#506] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LiuVII** [comment]
not sure if this is used, so opened an issue.
But really, I have a question
https://github.com/openkruise/kruise/issues/583

**Reply by FillZpp** [reply]
Sure, I have replied in the issue.
